pbrd/pbr_vty.h

```diff
@@ -20,7 +20,7 @@
 #ifndef __PBR_VTY_H__
 #define __PBR_VTY_H__
 
-#define PBR_STR "Policy based routing\n"
+#define PBR_STR "Policy Based Routing\n"
 
```

Comment:
proposal to move that definition to lib/,
because policy based routing term may be used too by BGP Flowspec.

There is a file pbr.h that could be used for that.
Or, I think command.h gathers those definitions.
any counter proposal welcome

Reply:
Most of the other XXX_STR are in command.h so reasonable request.